net/haproxy/src/opnsense/mvc/app/models/OPNsense/HAProxy/HAProxy.xml

```diff
@@ -9,6 +9,10 @@
                 <default>0</default>
                 <Required>Y</Required>
             </enabled>
+            <gracefulStop type="BooleanField">
+                <default>0</default>
+                <Required>Y</Required>
+            </gracefulStop>
```

Comment:
If this is required, you should bump the version number of the model to insert the new default right into the running config without the user going to the haproxy page :)